### PR TITLE
php72: migrate to homebrew/core

### PR DESCRIPTION
**Pull request by javian**
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
-----
I started on this PR since we will be merging php 7.2 in to core soon and at the same time retire it from the tap.

### CODE REVIEW

File: tap_migrations.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 {
   "phpenv": "homebrew/boneyard",
-  "phpsh": "homebrew/boneyard"
+  "phpsh": "homebrew/boneyard",
+  "php72": "homebrew/core/php",
```
Comment (SMillerDev) [inline tap_migrations.json]: Would this work? Don't we need an alias for that?

**Comment by benharold** [comment]
I can't find `php-gmp` anywhere now. According to [this page](http://www.braumeister.org/repos/Homebrew/homebrew-php/formula/php72-gmp) it was moved to `core` in this pull request, but when I try:

    brew search php-gmp

I get `No formula found for "php-gmp".`

**Comment by kabel** [comment]
@benharold That core extension is now built into the core `php` formula, no need to install anything separately to use it.

**Comment by klestoff** [comment]
@javian Is it possible to come back build options?

**Comment by javian** [comment]
@klestoff which build options are you referring to ? Most of the optional extensions that we had in the formula until now have been included by default in the new Formula and should no longer require options to enable them.

**Comment by klestoff** [comment]
@javian You know, I don't want some extensions and dependencies installed by default. For instance, httpd. :) On the other hand, I want to have an ability to build php with embed-sapi support. It was pretty handy when I could use options to build php.

**Comment by javian** [comment]
@klestoff its been discussed at length here https://github.com/Homebrew/homebrew-core/issues/24654 and it's not going to change.

**Comment by javian** [comment]
The embed sapi could of course be added to the Formula through a PR but the general idea is to not have any options - there should be one package that fits most or you would create your own formula.

**Comment by benharold** [comment]
Thanks @kabel, I ran `brew upgrade` and that took care of it.

**Comment by mdeboer** [comment]
@aurimasniekis it's already included and not available as extension anymore.

**Comment by MikeMcQuaid** [comment]
> For instance, httpd

This will not be installed any more if you use the bottle: https://github.com/Homebrew/homebrew-core/pull/24907

**Pull request merged**